collections/crowdsecurity/endlessh.yaml

```diff
@@ -0,0 +1,13 @@
+#the list of parsers it contains
+parsers:
+  - ./parsers/crowdsecurity/endlessh-logs.yaml
```

Comment:
in the collection itself the items should referenced by their name, in this case `crowdsecurity/endlessh-logs` and `crowdsecurity/endlessh-bf`

Reply:
@buixor Should I also update it in the `.tests/endlessh-logs/config.yaml`?

Reply:
sorry for the delay, looks good to me :)